### ISSUE TITLE
vTapioca

### ISSUE THREAD

**Issue by maximeseguin** [micmacIGN/micmac#39]
v1.0.beta11

Lors de l'utilisation de vTapioca le fichier MkTapioca n'est pas lancé par make.exe et j'ai le message d'erreur suivant :
 
`"C:/micmac/bin/mm3d" Pastis C:/Users/mseguin/Desktop/TestMicMac/ "NKS-Rel-AllCpleOfPattern@(DSC_2608.tif|DSC_2609.tif|DSC_2610.tif|DSC_2611.tif|DSC_2612.tif|DSC_2613.tif)" 2400  "MkF=C:/Users/mseguin/Desktop/TestMicMac/MkTapioca" NbMinPtsExp=2 ForceByDico=1 Detect=mm3d:Sift Match=mm3d:Ann  ratio=0.600000  NKS=NKS-Assoc-CplIm2Hom@@dat

make.exe:  C:/Users/mseguin/Desktop/TestMicMac/MkTapioca:  line 5:  Error: -- Multiply defined recipe for target C`

J'ai la même erreur lorsque je lance la commande directement dans la ligne de commande : 
`mm3d Tapioca All "C:/Users/mseguin/Desktop/TestMicMac/(DSC_2608.tif|DSC_2609.tif|DSC_2610.tif|DSC_2611.tif|DSC_2612.tif|DSC_2613.tif)" 2400`

Par contre tout fonctionne lorsque je lance la commande sans l'adressage absolu : 
`mm3d Tapioca All "(DSC_2608.tif|DSC_2609.tif|DSC_2610.tif|DSC_2611.tif|DSC_2612.tif|DSC_2613.tif)" 2400`

**Comment by howiie11** [comment]
I am struggling to solve this problem as well. Have you got any idea now?

**Comment by maximeseguin** [comment]
I've no idea. You should compare with older version because I'm working with the revision 6213 and there's no problem.

**Comment by howiie11** [comment]
OK, many thanks!

**Comment by howiie11** [comment]
Accidentally, I found that the cause for this error is that we used a absolute image path.

Change absolute path to relative path would make things work.

**Comment by maximeseguin** [comment]
Yes, but I don't know if you can force the graphical interface to work with a relative path.
